tests/make_models.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+
+"""Make simple test model in all tensorflow formats."""
```

Comment:
seems not directly used in test?
maybe we should consider put it under tools directory?